Makefile

```diff
@@ -9,13 +9,42 @@ GOTEST ?= go test $(if $V,-v)
 
 export GO111MODULE ?= on
 
+# Directories
+ROOT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+TOOLS_DIR := hack/tools
+TOOLS_BIN_DIR := $(abspath $(TOOLS_DIR)/bin)
+
+# Scripts
+GO_INSTALL := ./hack/go-install.sh
+
+# Binaries
+# Note: Need to use abspath so we can invoke these from subdirectories
+
+GOLANGCI_LINT_VER := v1.41.1
```

Comment:
Can we use this action instead of dealing with installing the bin here? https://github.com/golangci/golangci-lint-action